pegasus/sites.v3/hourofcode.com/views/how_to_nav.md

```diff
@@ -1,13 +1,17 @@
-- <a href="<%= resolve_url('/resources') %>"><%= hoc_s(:nav_back_to_resources)%></a>
-- <a href="<%= resolve_url('/resources/how-to') %>"><%= hoc_s(:how_to_nav_educators)%></a>
-- <a href="<%= resolve_url('/resources/how-to-public-officials') %>"><%= hoc_s(:how_to_nav_public_officials)%></a>
-- <a href="<%= resolve_url('/resources/how-to-districts') %>"><%= hoc_s(:how_to_nav_districts)%></a>
-- <a href="<%= resolve_url('/resources/how-to-events') %>"><%= hoc_s(:how_to_nav_assemblies)%></a>
-- <a href="<%= resolve_url('/resources/how-to-organizations') %>"><%= hoc_s(:how_to_nav_organizations)%></a>
+<div class="resources-navigation">

```

Comment:
I think we should make these navs haml files. It's looking pretty messy with all the tags. Plus anything in /views is really saying it should be haml.